ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/kinematicsToolboxModule/KinematicsToolboxController.java

```diff
@@ -781,7 +789,8 @@ protected boolean initializeInternal()
       else
       {
          initializePrivilegedConfiguration();
-         // It is required to update the tools now as it is only done at the end of each iteration.
+         // It is required to update the tools now as it is only done at the end of each
+         // iteration.
```

Comment:
Normally the formatter isn't supposed to format one-line comments. We'll have to look at that and maybe do a revision of the formatter.